tests/test_indices.py

```diff
@@ -13,275 +13,538 @@
     north_atlantic_oscillation,
     north_atlantic_oscillation_pc,
     north_atlantic_sea_surface_salinity,
+    sea_air_surface_temperature_anomaly_north_all,
+    sea_air_surface_temperature_anomaly_north_land,
+    sea_air_surface_temperature_anomaly_north_ocean,
+    sea_air_surface_temperature_anomaly_south_all,
+    sea_air_surface_temperature_anomaly_south_land,
+    sea_air_surface_temperature_anomaly_south_ocean,
     southern_annular_mode,
     southern_annular_mode_pc,
 )
 
 
-@pytest.mark.parametrize("source_name", list(VARNAME_MAPPING.keys()))
-def test_SAM_metadata(source_name):
-    """Ensure that index only contains time dimension."""
-    # Load test data
-    TEST_DATA_PATH = Path(__file__).parent / "../data/test_data/"
-    data_set = load_data_set(data_path=TEST_DATA_PATH, data_source_name=source_name)
+# @pytest.mark.parametrize("source_name", list(VARNAME_MAPPING.keys()))
+# def test_SAM_metadata(source_name):
+#     """Ensure that index only contains time dimension."""
+#     # Load test data
+#     TEST_DATA_PATH = Path(__file__).parent / "../data/test_data/"
+#     data_set = load_data_set(data_path=TEST_DATA_PATH, data_source_name=source_name)
+
+#     # Calculate SAM index
+#     SAM = southern_annular_mode(data_set)
+
+#     # Check, if calculated SAM index only has one dimension: 'time'
+#     assert SAM.dims[0] == "time"
+#     assert len(SAM.dims) == 1
+
+
+# @pytest.mark.parametrize("source_name", list(VARNAME_MAPPING.keys()))
+# def test_SAM_standardisation(source_name):
+#     """Ensure that standardisation works correctly."""
+#     # Load test data
+#     TEST_DATA_PATH = Path(__file__).parent / "../data/test_data/"
```

Comment:
I don't understand why there's so much code commented out here.

Reply:
Ah my bad, missed that and will correct it.
Just commented it out for the tests to run faster.
And the docstring is much better.

Reply:
Resolved with commit 9cbb69f0a3c925cabb885f5452ed5b7a77b630a2